tests/e2e/test_sampling_params.py

```diff
@@ -0,0 +1,355 @@
+import pytest
+from spyre_util import get_spyre_backend_list, get_spyre_model_list
+from vllm import LLM, SamplingParams
+
+
+@pytest.fixture(scope="module")
+@pytest.mark.parametrize("model", get_spyre_model_list())
+def spyre_model(model: str) -> LLM:
+    return LLM(model=model)
+
+
+@pytest.mark.parametrize("backend", get_spyre_backend_list())
+def test_spyre_batch1_temperature(spyre_model: LLM, backend: str,
+                                  monkeypatch: pytest.MonkeyPatch):
+    monkeypatch.setenv("VLLM_SPYRE_DYNAMO_BACKEND", backend)
+    monkeypatch.setenv("VLLM_SPYRE_OVERRIDE_SIGNALS_HANDLER", "1")
```

Comment:
If these patches are required for all functions, you can put them in a autouse fixture with module scope